app/views/assessments/_submission_panel.html.erb

```diff
@@ -15,7 +15,7 @@
   <div class="ui tab active" data-tab="upload" id="upload_tab">
     <div class="row handin-row">
       <div class="valign-wrapper drag-drop-handin" tabindex="0"
-          onclick="clickDrag();" onkeypress="clickDrag()" ondrop="dropHandler(event);" 
+          onclick="clickDrag();" onkeydown="clickDrag()" ondrop="dropHandler(event);"
```

Comment:
```suggestion
          onclick="clickDrag();" onkeydown="clickDrag(event)" ondrop="dropHandler(event);"
```
I think `event` needs to be added here to allow `clickDrag` to case on what key was pressed.

Reply:
Good catch -- fixed